en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,178 @@
+The VCC lines have to offer at least 3A continuous and should default to 5.1V. 
+A lower voltage of 5V is still acceptable, but discouraged.
+-->
+
+<!-- Data sheet for AP says 18.1 voltage divider, 36 Amps/volt 
+
+> **Note** Using the Power Module that comes with the kit you will need to configure the *Number of Cells* in the [Power Settings](https://docs.qgroundcontrol.com/en/SetupView/Power.html)
+> but you won't need to calibrate the *voltage divider*.
+> You will have to update the *voltage divider* if you are using any other power module (e.g. the one from the Pixracer).
+
+-->
+
+## Radio Control
+
+A remote control (RC) radio system is required if you want to *manually* control your vehicle (PX4 does not require a radio system for autonomous flight modes). 
+
+You will need to [select a compatible transmitter/receiver](../getting_started/rc_transmitter_receiver.md) and then *bind* them so that they communicate (read the instructions that come with your specific transmitter/receiver). 
+
+The instructions below show how to connect the different types of receivers to *Durandal*:
+
+
+- Spektrum/DSM receivers connect to the [DSM RC](../flight_controller/durandal.md#dsm-rc-port) input.
+  
+  ![Durandal - DSM](../../assets/flight_controller/durandal/dsm.jpg)
+
+- PPM and S.Bus receivers connect to the [SBUS_IN/PPM_IN](../flight_controller/durandal.md#rc-in) input port (marked as RC IN, next to the MAIN/AUX inputs).
+  
+  ![Durandal - Back Pinouts (Schematic)](../../assets/flight_controller/durandal/durandal_pinouts_back.jpg)
+
+- PPM and PWM receivers that have an *individual wire for each channel* must connect to the **PPM RC** port *via a PPM encoder* [like this one](http://www.getfpv.com/radios/radio-accessories/holybro-ppm-encoder-module.html) (PPM-Sum receivers use a single signal wire for all channels).
+
+For more information about selecting a radio system, receiver compatibility, and binding your transmitter/receiver pair, see: [Remote Control Transmitters & Receivers](../getting_started/rc_transmitter_receiver.md).
+
+
+## Telemetry Radios (Optional)
+
+Telemetry radios may be used to communicate and control a vehicle in flight from a ground station (for example, you can direct the UAV to a particular position, or upload a new mission).
+
+The vehicle-based radio should be connected to the [TELEM1](../flight_controller/durandal.md#telem1_2_3) port as shown below using one of the 6-pos connectors (if connected to this port, no further configuration is required).
+The other radio is connected to your ground station computer or mobile device (usually by USB).
+
+![Durandal/Telemetry Radio](../../assets/flight_controller/durandal/holybro_telemetry_radio.jpg)
+
+
+## SD Card (Optional)
+
+SD cards are most commonly used to [log and analyse flight details](../getting_started/flight_reporting.md).
+Insert a card into the *Durandal* where indicated below.
```

Comment:
I am assuming it comes with no card, as none shown in unboxing.

Reply:
I think it had one. There's typically one included, albeit not of best quality.

Reply:
Thanks. I will leave as is, because if it is present they will see it, and if not, they still know where to go get it.